docs/developer-guide/contributing.md

```diff
@@ -1,31 +1,24 @@
 # Contributing code
 
-Before contributing code, please familiarise yourself with the [Project conventions](/developer-guide/project-conventions.md).
+Before contributing code, please familiarise yourself with the [Project conventions](project-conventions.md).
 
 ## General advice:
```

Comment:
```suggestion
## General advice:

- Ensure the branch has the latest changes from the base branch as it may be out of sync between PR submission/rework/reviews. 
```
Add one at the beginning of these points. See also similar comment for `Merging branches`